util/smartbin_initialize.sql

```diff
@@ -11,12 +11,12 @@ CREATE TABLE devices(
     longitude DECIMAL(8,5)
 );
 
-CREATE TABLE distances(
+CREATE TABLE records(
     id INT PRIMARY KEY AUTO_INCREMENT,
     device_id INT,
     logged_datetime TIMESTAMP DEFAULT NOW(),
     distance DECIMAL(6,3) NOT NULL,
-    temperature INT, -- Are you sure you want an integer for temperature? How does the device store temperature data?
+    temperature INT,
```

Comment:
This gets read as a floating voltage from TMP36 and interpolated between the expected Celcius range. I'm not exactly sure how precise and accurate the sensor within 1 degree but it likely isn't that useful. Perhaps Decimal(3,1) is another alternative